src/storage.tf

```
@@ -242,3 +242,30 @@ resource "null_resource" "upload_tc_pdf" {
           EOT
   }
 }
+
+
+
+# Storage account to store backups: mainly api management
+module "backupstorage" {
+  count  = var.env_short == "p" ? 1 : 0
+  source = "git::https://github.com/pagopa/azurerm.git//storage_account?ref=v2.1.26"
+
+  name                     = replace(format("%s-backupstorage", local.project), "-", "")
+  account_kind             = "BlobStorage"
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
+  access_tier              = "Cool"
+  enable_versioning        = false
```

```suggestion
  enable_versioning        = true
```
for backup history... and maybe a policy to delete old blobs...